FrontEnd/src/components/basics/ImageDropdown/ImageDropdown.stories.tsx

```diff
@@ -1,14 +1,43 @@
 import React, { ComponentProps } from 'react';
 import { Story } from '@storybook/react';
 import ImageDropdown from './ImageDropdown';
+import Navbar from 'react-bootstrap/Navbar';
 
 export default {
   title: 'ImageDropdown',
   component: ImageDropdown,
 };
 
 const Template: Story<ComponentProps<typeof ImageDropdown>> = (args) => (
```

Comment:
This is the wrong usage of a template. A template should have only one instance of the component you want to render. Then each exported variable should be a "story" of that template to display the different use cases of the component you render inside the template. Consider using multiple templates instead or search on the storybook documentation for a better method to do this.

Reply:
resolved in an elegant way